services/minespace-web/src/components/common/RenderYear.js

```diff
@@ -79,6 +81,7 @@ export class RenderDate extends Component {
         onOpenChange={this.handleOpenChange}
         onPanelChange={this.handlePanelChange}
         onChange={this.clearValue}
+        disabledDate={this.props.disabledDate}
         disabled={this.props.disabled}
```

Comment:
Can the disabled flag be used here instead of creating a new disabled prop?

Reply:
`disabledDate` is a prop for `DatePicker`: https://ant.design/components/date-picker/

Reply:
Okay!